online-bookstore/src/main/java/group1/itschool/onlinebookstore/models/entities/UserEntity.java

```diff
@@ -2,15 +2,28 @@
 
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.persistence.Column;
 import jakarta.persistence.Entity;
 import jakarta.persistence.Table;
 import lombok.Data;
 
 
-//@Data
-//@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
-//@Entity
-//@Table(name = "users")
-//public class UserEntity {
-//
-//}
\ No newline at end of file
+@Data
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+@Entity
+@Table(name = "users")
```

Comment:
We might need an unique constraint for the 'username' field. This way we are assured that there's no duplicate users in the users table.